aptos-move/framework/aptos-framework/sources/transaction_validation.spec.move

```diff
@@ -1,483 +1,484 @@
-        let transaction_fee_amount = txn_gas_price * gas_used;
-
-        // Check account invariants.
-        let addr = signer::address_of(account);
-        // TODO(fa_migration)
-        // let pre_balance = global<coin::CoinStore<AptosCoin>>(gas_payer).coin.value;
-        // let post balance = global<coin::CoinStore<AptosCoin>>(gas_payer).coin.value;
-        let pre_account = global<account::Account>(addr);
-        let post account = global<account::Account>(addr);
-
-        aborts_if !exists<CoinStore<AptosCoin>>(gas_payer);
-        aborts_if !exists<Account>(addr);
-        aborts_if !(global<Account>(addr).sequence_number < MAX_U64);
-        // aborts_if pre_balance < transaction_fee_amount;
-        // ensures balance == pre_balance - transaction_fee_amount + storage_fee_refunded;
-        ensures account.sequence_number == pre_account.sequence_number + 1;
-
-        // Check burning.
-        //   (Check the total supply aggregator when enabled.)
-        let amount_to_burn = transaction_fee_amount - storage_fee_refunded;
-        let apt_addr = type_info::type_of<AptosCoin>().account_address;
-        let maybe_apt_supply = global<CoinInfo<AptosCoin>>(apt_addr).supply;
-        let total_supply_enabled = option::spec_is_some(maybe_apt_supply);
-        let apt_supply = option::spec_borrow(maybe_apt_supply);
-        let apt_supply_value = optional_aggregator::optional_aggregator_value(apt_supply);
-        let post post_maybe_apt_supply = global<CoinInfo<AptosCoin>>(apt_addr).supply;
-        let post post_apt_supply = option::spec_borrow(post_maybe_apt_supply);
-        let post post_apt_supply_value = optional_aggregator::optional_aggregator_value(post_apt_supply);
-
-        aborts_if amount_to_burn > 0 && !exists<AptosCoinCapabilities>(@aptos_framework);
-        aborts_if amount_to_burn > 0 && !exists<CoinInfo<AptosCoin>>(apt_addr);
-        aborts_if amount_to_burn > 0 && total_supply_enabled && apt_supply_value < amount_to_burn;
-        ensures total_supply_enabled ==> apt_supply_value - amount_to_burn == post_apt_supply_value;
-
-        // Check minting.
-        let amount_to_mint = storage_fee_refunded - transaction_fee_amount;
-        let total_supply = coin::supply<AptosCoin>;
-        let post post_total_supply = coin::supply<AptosCoin>;
-
-        aborts_if amount_to_mint > 0 && !exists<CoinStore<AptosCoin>>(addr);
-        aborts_if amount_to_mint > 0 && !exists<AptosCoinMintCapability>(@aptos_framework);
-        aborts_if amount_to_mint > 0 && total_supply + amount_to_mint > MAX_U128;
-        ensures amount_to_mint > 0 ==> post_total_supply == total_supply + amount_to_mint;
-
-        let aptos_addr = type_info::type_of<AptosCoin>().account_address;
-        aborts_if (amount_to_mint != 0) && !exists<coin::CoinInfo<AptosCoin>>(aptos_addr);
-        include coin::CoinAddAbortsIf<AptosCoin> { amount: amount_to_mint };
-    }
-}
+// spec aptos_framework::transaction_validation {
```

Comment:
There is no need to comment this out. See https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/FRAMEWORK-PROVER-GUIDE.md on how to turn off a spec. @vusirikala